theano/gpuarray/tests/test_linalg.py

```diff
@@ -214,7 +214,7 @@ def test_gpu_matrix_inverse(self):
         N = 1000
```

Comment:
tests should set the seed of the theano.tests.unittests_tools.seed_rng() before calling np.random* function to have the test determinisctic. The call rand() bellow call np.random*.

seed_rng() will use the seed by default from the flag unittests.rseed. So you can easily run the tests with many seed to see if it pass frequently enough:

for i in `seq 1 100`; do echo $i; export THEANO_FLAGS=unitests.rseed=$i; nosetests ...; done